scripts/add-commit-hash-to-env.sh

```diff
@@ -0,0 +1,19 @@
+if [ -f ".env" ]
```

Comment:
Aren't you missing `#!/bin/bash` in top of the file?

Reply:
Yep, will fix!